dspace-server-webapp/src/main/java/org/dspace/app/rest/signposting/relation/ItemTypeProcessor.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.signposting.relation;
+
+import java.text.MessageFormat;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.dspace.app.rest.signposting.model.Linkset;
+import org.dspace.app.rest.signposting.model.Lset;
+import org.dspace.app.rest.signposting.model.Relation;
+import org.dspace.app.rest.signposting.processor.ItemSignPostingProcessor;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataFieldName;
+import org.dspace.content.service.ItemService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * An implementation of {@link ItemSignPostingProcessor} for the type relation.
+ */
+public class ItemTypeProcessor extends ASignPostingProcessor
```

Comment:
"type" relation provides a first link that has as target the [schema.org](https://schema.org/) term (HTTP URI) for a [Creative Work](https://schema.org/CreativeWork) that best characterizes the scholarly object as a whole. For example, provide https://schema.org/ScholarlyArticle in case the object is a scholarly paper; provide https://schema.org/Dataset if it is a dataset. If no appropriate term is available in https://schema.org/, select one (HTTP URI) from a commonly used ontology.
Provide a second link that has as target https://schema.org/AboutPage in the common case where an actual landing page is concerned.